### PR TITLE
comment

### PR DESCRIPTION
**Pull request by Jasmine-Bascom**

### CODE REVIEW

**Comment by CompBioJasmine** [comment]
fixrbot inspect 1

**Comment by CompBioJasmine** [comment]
fixrbot inspect 1

**Comment by Jasmine-Bascom** [comment]
fixrbot inspect 1

**Comment by CompBioJasmine** [comment]
> > fixrbot inspect 1
> 
> Mismatch pattern in method `userList`
> 
> Interactions:
> 
> * `fixrbot pattern`: Gets the detailed information of the pattern
> * `fixrbot examples`: Gets the example code of the pattern

fixrbot pattern

File: app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
  * The main motive is to abstract away all the nitty gritty database
  * stuff for simpler cleaner high level access.
  */
```
[the remark this answers is not in the document]
Comment (CompBioJasmine) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java]: fixrbot pattern

[the remark this answers is not in the document]
Comment (CompBioJasmine) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java]: fixrbot pattern

[the remark this answers is not in the document]
Comment (CompBioJasmine) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java]: fixrbot example

[the remark this answers is not in the document]
Comment (Jasmine-Bascom) [reply app/src/main/java/org/developfreedom/logmein/DatabaseEngine.java:41]: fixrbot pattern

**Pull request closed without merging**